src/types.ts

```diff
@@ -0,0 +1,13 @@
+import mmdb = require('maxmind');
+
+export interface IMaxMind {
+  queries_remaining: number;
+}
+
+export interface ICountryResponse extends mmdb.ICountryResponse {
+  maxmind?: IMaxMind;
+}
+
+export interface ICityResponse extends mmdb.ICityResponse {
```

Comment:
This *could* extend `ICountryResponse {}` I think. Would align with`City` extending `Country`  on the class level

Reply:
Ah, nevermind. I got mixed up. Ignore this.